duckling/duckling.py

```diff
@@ -78,10 +78,33 @@ def _create_classpath(self):
                     jars.append(os.path.join(top, file_name))
         return os.pathsep.join(jars)
 
-    def load(self):
-        """Loads the Duckling corpus"""
+    def load(self, languages=None):
+        """Loads the Duckling corpus.
+
+        Languages can be specified, defaults to all.
+
+        Args:
+            languages: Optional parameter to specify languages,
+                e.g. [Duckling.ENGLISH, Duckling.FRENCH] or supported ISO 639-1 Codes (e.g. ["en", "fr"])
+        """
         duckling_load = self.clojure.var("duckling.core", "load!")
-        duckling_load.invoke()
+
+        if languages is not None:
+            # Duckling's load function expects ISO 639-1 Language Codes (e.g. "en")
+            languages = [Language.convert_to_iso(lang) for lang in languages]
+
+            clojure_hashmap = self.clojure.var("clojure.core", "hash-map")
```

Comment:
`clojure_hashmap` , `clojure_list` can be. moved up to line 90, so it's after `duckling_load`